cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -377,8 +381,22 @@ func (s *VMTServer) Run() {
 		// AUTOMEMLIMIT_DEBUG environment variable enables debug logging of AUTOMEMLIMIT
 		_ = os.Setenv("AUTOMEMLIMIT_DEBUG", "true")
 		memlimit.SetGoMemLimitWithEnv()
+		if s.ListQueryItemsCount == 0 {
+			limit, err := memlimit.FromCgroup()
+			if err != nil || limit == 0 {
+				// This is very unlikely because in absense of any limit set in container resources
+				// we will get the cgroup limit as the nodes available/usable memory limit
+				glog.Errorf("Error retrieving memory limit (%v) or limit is not set: %v.", limit, err)
+			} else {
+				glog.V(2).Infof("Cgroup memlimit is set as %v.", limit)
+				util.ListQueryItemsCount = calculateListAPIItemsCount(float64(limit))
+			}
+		} else {
+			util.ListQueryItemsCount = s.ListQueryItemsCount
```

Comment:
Do we want to do some validation on this number (negative value, for example)? Maybe set some minimum value?